src/Bridge/Symfony/Validator/Metadata/Property/ValidatorPropertyMetadataFactory.php

```diff
@@ -111,33 +117,64 @@ public function create(string $resourceClass, string $name, array $options = [])
                     $iri = self::SCHEMA_MAPPED_CONSTRAINTS[\get_class($constraint)] ?? null;
                 }
 
-                if (null !== $required && null !== $iri) {
-                    break 2;
+                foreach ($this->restrictionsMetadata as $restrictionMetadata) {
+                    if ($restrictionMetadata->supports($constraint, $propertyMetadata)) {
+                        $restrictions[] = $restrictionMetadata->create($constraint, $propertyMetadata);
+                    }
                 }
             }
         }
 
-        return $propertyMetadata->withIri($iri)->withRequired($required ?? false);
+        $propertyMetadata = $propertyMetadata->withIri($iri)->withRequired($required ?? false);
```

Comment:
We don't break the above loop anymore, and it looks like we can have multiple constraints, hence different iris? I'm not sure that this case is handled / implemented correctly.

Reply:
@soyuka Like in previous implementation we continue to set `IRI` and `required` only if it is not set before.